### PR TITLE
Restore positioning of legacy action menu

### PR DESCRIPTION
**Pull request by tilt**
https://www.openproject.org/work_packages/8210

### CODE REVIEW

File: app/assets/stylesheets/content/_legacy_actions.sass [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * See doc/COPYRIGHT.rdoc for more details.  ++
+ */
+
+@mixin legacy_actions_defaults($margin-top: 7px)
```
Comment (myabc) [inline app/assets/stylesheets/content/_legacy_actions.sass:29]: not sure if this comes from code, but you can use dashes in mixin names now.

**Comment by tilt** [comment]
seems to work for the plugins as well

File: app/assets/stylesheets/content/_legacy_actions.sass (hunk 1 above)
Comment (tilt) [inline app/assets/stylesheets/content/_legacy_actions.sass:29]: I wasn't too perfectionistic about this

**Comment by myabc** [comment]
This breaks tests though :(

**Comment by tilt** [comment]
yeah sure, 'cause they use the former class names as selectors :dancer:

**Comment by tilt** [comment]
All the specs in the plugins don't match the css class by name - the selectors by '.action_menu_specific' should work on the other hand so I think after Travis says ok it's safe to merge

**Comment by tilt** [comment]
@myabc we can now "safely" merge this

**Pull request merged**